src/base.css

```diff
@@ -1177,21 +1171,37 @@ Possible extra rowspan handling
 
 	.toc a {
 		/* More spacing; use padding to make it part of the click target. */
-		padding-top: 0.1rem;
+		padding: 0.1rem 1px 0;
 		/* Larger, more consistently-sized click target */
 		display: block;
+		/* Switch to using border-bottom */
+		text-decoration: none;
+		border-bottom: 1px solid;
+		border-bottom: 1px solid;
 		/* Reverse color scheme */
 		color: black;
 		color: var(--toclink-text);
+		text-decoration-color: #3980b5;
+		text-decoration-color: var(--toclink-underline);
 		border-color: #3980b5;
 		border-color: var(--toclink-underline);
 	}
 	.toc a:visited {
 		color: black;
 		color: var(--toclink-visited-text);
+		text-decoration-color: #054572;
+		text-decoration-color: var(--toclink-visited-underline);
 		border-color: #054572;
 		border-color: var(--toclink-visited-underline);
 	}
+	.toc a:focus,
+	.toc a:hover {
+		background: #f8f8f8;
```

Comment:
We don't generally try to fallback for UAs that don't support *rgba()*; at least, the other 8 places we use the function don't have a hex fallback.